File: django_apollo/blog/types/comment_type.py
import graphene
from graphene.contrib.django import DjangoObjectType

from blog.models import Comment


class CommentType(graphene.ObjectType):
    id = graphene.String()
    body = graphene.String(description='body of the comment')
    user = graphene.Field(
        'usertype',
        id=graphene.ID(),
        description='User for this comment',
    )

File: ruby/rails_graphql/app/api/types/comment_type.rb
CommentType = GraphQL::ObjectType.define do
  name "Comment"
  description "A comment"

  interfaces [NodeIdentification.interface]
  global_id_field :id

  field :body, !types.String
  field :user, -> { UserType }
  field :post, -> { PostType }
end
